gameLogic/GameMap.java

```diff
@@ -9,133 +9,108 @@
-					} else if (w == 9){
-						mapOne[h][w] = new ImageIcon("src/assets/maps/background10.png");
-						mapTwo[h][w] = new ImageIcon("src/assets/maps/background10.png");
-						mapThree[h][w] = new ImageIcon("src/assets/maps/background10.png");
-					} else {
-						mapOne[h][w] = new ImageIcon("src/assets/maps/background6.png");
-						mapTwo[h][w] = new ImageIcon("src/assets/maps/background6.png");
-						mapThree[h][w] = new ImageIcon("src/assets/maps/background6.png");
-					}
-				} else if (w == 0) {
-					mapOne[h][w] = new ImageIcon("src/assets/maps/background4.png");
-					mapTwo[h][w] = new ImageIcon("src/assets/maps/background4.png");
-					mapThree[h][w] = new ImageIcon("src/assets/maps/background4.png");
-				} else if (w == 9) {
-					mapOne[h][w] = new ImageIcon("src/assets/maps/background7.png");
-					mapTwo[h][w] = new ImageIcon("src/assets/maps/background7.png");
-					mapThree[h][w] = new ImageIcon("src/assets/maps/background7.png");
-				}
+		for (int i = 0; i < TOTAL_LEVELS; i++) {
+			for (int j = 0; j < MAPS_PER_LEVEL; j++) {
+				hitRects[i][j] = new ArrayList<>();
+			}
+		}
 
-				if (((h > 1 && h < 4) || (h > 5 && h < 8)) && ((w > 1 && w < 4) || (w > 5 && w < 8))) {
-					mapOne[h][w] = new ImageIcon("src/assets/maps/background2.png");
-					hitRectOne.add(new Rectangle(w * 100, h * 100, 100, 100));
-				} else if (w != 0 && w != 9 && h != 9 && h != 0) {
-					mapOne[h][w] = new ImageIcon("src/assets/maps/background1.png");
+		for (int i = 0; i < TOTAL_LEVELS; i++) {
+			for (int m = 0; m < NUM_TILE_TYPES; m++) mapTiles[m] = new ImageIcon(FILEPATH_ROOT+
+					(i+1)+FILEPATH_BACKGROUND+m+FILEPATH_PNG);
+			for (int j = 0; j < MAPS_PER_LEVEL; j++) {
+				maps[i][j][0][0] = mapTiles[1];
+				maps[i][j][0][9] = mapTiles[2];
+				maps[i][j][7][0] = mapTiles[3];
+				maps[i][j][7][9] = mapTiles[4];
+				for (int w = 1; w < MAP_WIDTH - 1; w++) {
+					maps[i][j][0][w] = mapTiles[5];
+					maps[i][j][7][w] = mapTiles[6];
 				}
-
-				if ((h > 2 && h < 7) && (w > 2 && w < 7)) {
-					mapTwo[h][w] = new ImageIcon("src/assets/maps/background2.png");
-					hitRectTwo.add(new Rectangle(w * 100, h * 100, 100, 100));
-				} else if (w != 0 && w != 9 && h != 9 && h != 0) {
-					mapTwo[h][w] = new ImageIcon("src/assets/maps/background1.png");
+				for (int h = 1; h < MAP_HEIGHT - 1; h++) {
+					maps[i][j][h][0] = mapTiles[7];
+					maps[i][j][h][9] = mapTiles[8];
 				}
-
-				if (w != 0 && w != 9 && h != 9 && h != 0) {
-					mapThree[h][w] = new ImageIcon("src/assets/maps/background1.png");
+				for (int h = 0; h < MAP_HEIGHT - 2; h++) {
+					for (int w = 0; w < MAP_WIDTH - 2; w++) {
+						if (GameMapPresets.LEVEL_MAPS[i][j][h][w] == 1) {
+							maps[i][j][h + 1][w + 1] = mapTiles[9];
+							hitRects[i][j].add(new Rectangle((w+1)*TILE_LENGTH, (h+1)*TILE_LENGTH, TILE_LENGTH, TILE_LENGTH));
+						} else {
+							maps[i][j][h + 1][w + 1] = mapTiles[0];
+						}
```

Comment:
I think a better way to do this is to include the border as a number in the LEVEL_MAPS array, then simply take the number from the array, find the mapTile[x] and assign. We can save a lot of for loops this way. For hitRects, we just watch for specific numbers to show up, eg we can designate background9 for solid squares.